sbt-scalajs-cross/src/main/scala/scalajscross/ScalaJsCross.scala

```diff
@@ -0,0 +1,41 @@
+package scalajscross
+
+import sbtcross._
+
+import sbt._
+
+import org.scalajs.sbtplugin.{ScalaJSPlugin, ScalaJSCrossVersion}
+
+case object JSPlatform extends Platform {
```

Comment:
Let's leave it in this repo for now. Before sbt-cross-project turns 1.0.0, it should be moved out, though. Either enter `scala-js/scala-js` or be in its own separate repo.